be/src/runtime/runtime_state.cpp

```diff
@@ -380,6 +367,22 @@ void RuntimeState::append_error_msg_to_file(const std::string& line, const std::
         out << "Error: " << error_msg << ". Row: " << line;
     }
 
+    // If file havn't been opened, open it here
+    if (_error_log_file == nullptr) {
+        Status status = create_error_log_file();
+        if (!status.ok()) {
+            LOG(WARNING) << "Create error file log failed. because: " << status.get_error_msg();
+            if (_error_log_file != nullptr) {
```

Comment:
looks like this `_error_log_file` is perfectly to be managed by std::unique_ptr.